src/openrct2-cli/Cli.cpp

```diff
@@ -20,7 +20,7 @@ using namespace OpenRCT2;
 int main(int argc, const char** argv)
 {
     int32_t rc = EXIT_SUCCESS;
-    int runGame = cmdline_run(argv, argc);
+    int runGame = CmdLineRun(argv, argc);
```

Comment:
My refactor, as well as the file name, use a lower case `l`.
I don’t mind one or the other, or indeed `CommandLine` (which is even better).

Reply:
I agree with the full name, but let's leave that for later.

Reply:
https://github.com/OpenRCT2/OpenRCT2/issues/19176